checks.d/iis.py

```diff
@@ -87,6 +95,10 @@ def check(self, instance):
                     filters=filters, instance_tags=instance_tags
                 )
             )
+        except pythoncom.com_error:
```

Comment:
We could be a little bit more specific here and log a warning only when the error string contains the error code that we expect, i.e.:

```python
except pythoncom.com_error as e:
    if '0x80041017' in str(e):
        self.warning("blah blah")
    raise e
```

Also, we could use `self.warning` instead of `self.log.warning`.

Thoughts?